.github/dependabot.yml

```diff
@@ -2,11 +2,15 @@
 
 version: 2
 updates:
-  - package-ecosystem: "maven"
-    directory: "/"
-    schedule:
-      interval: "daily"
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "daily"
+# Maintain dependencies for your plugin
+- package-ecosystem: maven
+  directory: /
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Comment:
This line ensures that at the time of the dependency update process, the number of pull requests should not be more than 10. It helps manage resources such as CI/CD pipelines and review capacity.